### DISCUSSION TITLE
cn/2017/12/evd-svd-and-pca/

### DISCUSSION THREAD

**Discussion by giscus** [leovan/leovan.me#27] [Comments]
# cn/2017/12/evd-svd-and-pca/

准备知识 向量与基 $\renewcommand{\diag}{\operatorname{diag}}\renewcommand{\cov}{

https://leovan.me/cn/2017/12/evd-svd-and-pca/

**Comment by linnnnnnnda** [comment]
hello  或许公式（4）是不是写反了呢；以及，是主成分分析吧

**Reply by leovan** [reply]
感谢，已修正
